### DISCUSSION TITLE
When exporting a paper title containing TeX, it always fails to compile properly in latex

### DISCUSSION THREAD

**Discussion by meatless-tim** [retorquere/zotero-better-bibtex#2683] [Q&A] [answered]
This is the paper title in Zotero `Phase transitions in a non-Hermitian Aubry-Andr\'e-Harper model`, when I use Better BibTex to export, it will be `title = {Phase Transitions in a Non-{{Hermitian Aubry-Andr}}\textbackslash 'e-{{Harper}} Model}` in the .bib file.

When I try to compile the .bib in latex, the outcome is

```
Longhi S. Phase transitions in a non-Hermitian Aubry-Andr\’e-Harper model[J/OL].
Physical Review B, 2021, 103(5):054203. DOI: 10.1103/PhysRevB.103.054203."
```

I have to revise the  `title = {Phase Transitions in a Non-{{Hermitian Aubry-Andr}}\textbackslash 'e-{{Harper}} Model}` to be  `title = {Phase Transitions in a Non-{{Hermitian Aubry-Andr\'e}}-{{Harper}} Model}` in the .bib file.

How can I fix this problem?

**Comment by retorquere** [comment, marked as answer]
Replace `Andr\'e` with `André` in Zotero. Zotero does not understand TeX.

**Comment by meatless-tim** [comment]
The title of the article was synchronized from the web page using the
zotero connector, so do I have to modify it manually?

Emiliano Heyns ***@***.***>于2023年10月23日 周一01:17写道：

**Reply by retorquere** [reply]
Yes. You can also report the problem on the forum, the import translator should have replaced that.

**Comment by meatless-tim** [comment]
Hi, there is no any debug log.

Emiliano Heyns ***@***.***> 于2023年10月22日周日 23:25写道：
